smrf/envphys/radiation_ipw.py

```diff
@@ -0,0 +1,168 @@
+import os
+import subprocess as sp
+
+import numpy as np
+on_rtd = os.environ.get('READTHEDOCS') == 'True'
+if on_rtd:
+    IPW = '.'  # placehold while building the docs
+elif 'IPW' not in os.environ:
+    IPW = '/usr/local/bin'
+else:
+    IPW = os.environ['IPW']     # IPW executables
```

Comment:
It could probably be done in setup.py? The IPW executable aren't always on the path so it makes it tough and subprocess needs to know that env var. There are now only 2 calls after this PR to IPW, so this could be removed at some point.